docs/guides/dws-on-csm/readme.md

```diff
@@ -0,0 +1,258 @@
+yq w -i slurm-config-templates.yaml 'data."slurm.conf"' "$(cat slurm.conf)"
+```
+
+Add the `burst_buffer.lua` plugin script and `burst_buffer.conf` configuration file to the ConfigMap.
+
+```console title="ncn-m001:~ #"
+yq w -i slurm-config-templates.yaml -- 'data."burst_buffer.lua"' "$(cat dws-slurm-bb-plugin/src/burst_buffer/burst_buffer.lua)"
+yq w -i slurm-config-templates.yaml 'data."burst_buffer.conf"' "$(cat dws-slurm-bb-plugin/src/burst_buffer/burst_buffer.conf)"
+```
+
+Apply the updated ConfigMap resource.
+
+```console title="ncn-m001:~ #"
+kubectl apply -f slurm-config-templates.yaml
+```
+
+### Restart the Slurm Configuration Job
+
+The Slurm configuration job will process the ConfigMap into a second one that is included in the pod spec.  The following steps to reconfigure Slurm are found in the HPE Cray Programming Installation Guide.
+
+```console title="ncn-m001:~ #"
+JOBNAME=$(kubectl get job -n services | grep slurm-config | grep -v import | awk '{print $1}')
+kubectl get job -n services -o yaml $JOBNAME > slurm-config.yaml
+kubectl delete -f slurm-config.yaml
+yq d -i slurm-config.yaml spec.template.metadata
+yq d -i slurm-config.yaml spec.selector
+kubectl apply -f slurm-config.yaml
+SLURMCTLD_POD=$(kubectl get pod -n user -lapp=slurmctld -o jsonpath='{.items[0].metadata.name}')
+kubectl exec -n user $SLURMCTLD_POD -c slurmctld -- scontrol reconfigure
+```
+
+After the configuration job has been restarted, wait about 30 seconds to see the new content appear inside the pod.
+
+If you were only adding or modifying burst_buffer.lua, then Slurm is now ready for your jobs.  If you were also modifying slurm.conf to set the BurstBufferType, or you were adding or modifying burst_buffer.conf, then you must also restart the Slurm deployments.
+
+```console title="ncn-m001:~ #"
+kubectl rollout restart deployment -n user slurmctld-backup
+kubectl wait deployment --timeout=60s -n user --for condition=Available=True -l app=slurmctld-backup
+kubectl rollout restart deployment -n user slurmctld
+kubectl wait deployment --timeout=60s -n user --for condition=Available=True -l app=slurmctld
+```
+
+## Submit a test job
+
+Use a UAN host to submit a test job.  The following assumes a UAN host named `uan01` and a user account named `vers`.
+
+```console
+ncn-m001:~ # ssh uan01
+uan01:~ # su vers
+vers@uan01:/root> cd /lus/vers
+```
+
+Create the following example test job:
+
+```conf title="vers@uan01:/lus/vers> cat /tmp/dws-test" linenums="1"
+#!/bin/sh                
+#SBATCH --time=1
+#DW
+/bin/hostname
+srun -l /bin/hostname
+srun -l /bin/pwd
+```
+
+Note that we will submit the test job from a directory where our `vers` account can write its output files.
+
+```console title="vers@uan01:/lus/vers>"
+sbatch /tmp/dws-test
+```
+
+The sbatch command will tell you the ID of your job.
+```console
+Submitted batch job 7773
+```
+
+You can use that to query the job's status.  The job's output will be in the file named `slurm-<ID>`.
```

Comment:
>  The job's output will be in the file named `slurm-<ID>`

The job's output file is configurable using the `sbatch` command or `sbatch` directive in the job script.

Reply:
```suggestion
You can use that to query the job's status.  In this example, the job's output will be in the file named `slurm-<ID>`.
```

Reply:
That's good, I'll add that.